coding/string_utf8_multilang.cpp

```diff
@@ -89,23 +89,38 @@ constexpr bool IsSupportedLangCode(int8_t langCode)
 }
 }  // namespace
 
+bool StringUtf8Multilang::IsServiceLang(std::string_view lang)
+{
+  return lang == kLanguages[kDefaultCode].m_code
+      || lang == kLanguages[kInternationalCode].m_code
+      || lang == kLanguages[kAltNameCode].m_code
+      || lang == kLanguages[kOldNameCode].m_code;
+}
 // static
-StringUtf8Multilang::Languages const & StringUtf8Multilang::GetSupportedLanguages()
+StringUtf8Multilang::Languages const & StringUtf8Multilang::GetSupportedLanguages(bool returnServiceLanguages)
 {
   // Asserts for generic class constants.
   ASSERT_EQUAL(kLanguages[kDefaultCode].m_code, std::string_view{"default"}, ());
   ASSERT_EQUAL(kLanguages[kInternationalCode].m_code, std::string_view{"int_name"}, ());
   ASSERT_EQUAL(kLanguages[kAltNameCode].m_code, std::string_view{"alt_name"}, ());
   ASSERT_EQUAL(kLanguages[kOldNameCode].m_code, std::string_view{"old_name"}, ());
   ASSERT_EQUAL(kLanguages[kEnglishCode].m_code, std::string_view{"en"}, ());
-  static StringUtf8Multilang::Languages languages;
-  if (languages.empty())
+  static StringUtf8Multilang::Languages allLanguages;
+  static StringUtf8Multilang::Languages languagesWithoutService;
```

Comment:
Can these variables be made static const, global (above the function), and initialized by lambdas?